src/AirtableClient.php

```diff
@@ -123,6 +126,25 @@ public function add(string $table, array $fields, ?string $dataClass = null): ?A
         return AirtableRecord::createFromRecord($recordData);
     }
 
+    public function createForm(array $fields): FormInterface
+    {
+        $formFactory = Forms::createFormFactoryBuilder()
```

Comment:
Here we should have something like `if(class_exists(FormInterface)` or something like that to check that Form component is installed instead of forcing installation via composer.json :)

Reply:
Do you suggest simply informing the user that they should add this dependency, rather than adding it systematically?
Indeed, for the case of a user not using forms, it is one less dependency in his project.

I hesitate between the sobriety of the dependencies of the bundle, and the simplicity of use ;-)

Reply:
```php
throw new \Exception("The AirtableClient::createForm method needs the Symfony Form Component to be installed, please add it with Composer : composer req form");
```

And voilààà ! :p

Reply:
And thanks frère ;-)